### ISSUE TITLE
Add support for Windows Package Manager (winget)

### ISSUE THREAD

**Issue by IDisposable** [Genymobile/scrcpy#1444]
- [X] I have checked that a similar [feature request](https://github.com/Genymobile/scrcpy/issues?q=is%3Aopen+is%3Aissue+label%3A%22feature+request%22) does not already exist.

**Is your feature request related to a problem? Please describe.**
While `Chocolatey` and `Scoop` are both excellent package managers for `Windows`, the newly released `Windows Package Manager` is [built-in and available in all upcoming versions of Windows](https://devblogs.microsoft.com/commandline/windows-package-manager-preview/).

**Describe the solution you'd like**
Would love to see scrcpy registered for a `winget install scrcpy` install. Instructions for [submitting the package to the winget repository](https://docs.microsoft.com/en-us/windows/package-manager/package/repository) are pretty simple, just uploading a [manifest](https://docs.microsoft.com/en-us/windows/package-manager/package/manifest?tabs=minschema%2Ccompschema) to the [public GitHub repository of packages](https://github.com/microsoft/winget-pkgs) via a PR.

Steps:
1. Create manifest
1. Register manifest with winget
1. Update the srccpy repository to build/validate/submit manifest
1. Wait for initial confirmation that package is available
1. Update this repository README.MD to include winget instructions.

**Describe alternatives you've considered**
Obviously we can already use [choco](https://chocolatey.org/) or [scoop](https://scoop.sh/), but those package managers have to be bootstrap-installed first if not yet in use.

**Comment by IDisposable** [comment]
@rom1v have you typically been the person submitting the `choco` or `scoop` manifests? Would you like me to create a sample, or just submit one. Would you like me to create the PR for the README.MD file?

**Comment by rom1v** [comment]
> have you typically been the person submitting the `choco` or `scoop` manifests?

No, people just submitted pull requests to add the links to the README (#1144 #1202).

> Would you like me to create a sample, or just submit one. Would you like me to create the PR for the README.MD file?

Yes. Yes. :wink:

Thank you :+1:

**Comment by AzureRaptor** [comment]
I very much second this.

It's _about time_ Windows got an "official" package manager - all the rest of them have various issues or taints, and this is a pretty darn good implementation.  

I just checked the repository a little while ago and I didn't see scrcpy in there.  @IDisposable, thank you for spearheading this! Have you submitted the manifest yet? Please let us know if/when there's been any response.

**Comment by kunal26das** [comment]
When can we expect windows terminal cli winget package of scrcpy?

**Comment by AzureRaptor** [comment]
Indeed, it's been quite some time since there has been any movement on this
issue.

On Sun, Mar 7, 2021 at 6:26 PM Kunal Das - notifications@github.com
<gitmaximus.spamviking.0c93c2b35d.notifications#reply@reply.github.com>
wrote:

**Comment by hazre** [comment]
Any update on this?

**Comment by Hazmi35** [comment]
> Any update on this?

This is blocked by microsoft/winget-cli#140, also related: microsoft/winget-pkgs#4083

**Comment by vignesh-em** [comment]
Is this actively being looked into?

**Comment by nevubm** [comment]
> This is blocked by [microsoft/winget-cli#140](https://github.com/microsoft/winget-cli/issues/140), also related: [microsoft/winget-pkgs#4083](https://github.com/microsoft/winget-pkgs/issues/4083)

Looks like https://github.com/microsoft/winget-cli/issues/140 has been fixed. [Release notes](https://github.com/microsoft/winget-cli/releases/tag/v1.4.2161-preview) indicate installing from zip is currently experimental and the Community Repository will not accept zip applications until 1.4 is Generally Available, but it's coming

**Comment by davvolun** [comment]
I believe a stable (non-prerelease) version of Winget 1.4 is available, as of about 3 weeks ago https://github.com/microsoft/winget-cli/releases/tag/v1.4.10173

**Comment by AzureRaptor** [comment]
> I believe a stable (non-prerelease) version of Winget 1.4 is available, as of about 3 weeks ago https://github.com/microsoft/winget-cli/releases/tag/v1.4.10173

@IDisposable  Does this mean you can go ahead with submitting that manifest/PR now?

**Comment by IDisposable** [comment]
I guess it does! I'll get to that on Thursday.

**Comment by rom1v** [comment]
#4005

**Comment by davvolun** [comment]
I'm guessing this should be added as a separate issue, but this didn't add scrcpy or, a bigger problem, adb to my command line PATH. It installed them, but I expected to be able to simply run `adb connect <android ip>` and `scrcpy` to view my screen. Physically connecting with scrcpy works fine, but without (easy) command line access as part of the package, wireless connection for adb is a pain.

Was this as expected, a bug, or a fault on my end?
